src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -432,7 +465,20 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
      * @return {void}
      */
     fetchFeedItemsSuccessCallback = (feedItems: FeedItems): void => {
-        this.setState({ feedItems, activityFeedError: undefined });
+        const {
+            annotatorState: { error },
+        } = this.props;
+
+        let newFeedItems = feedItems;
+
+        if (error && error.code === ERROR_CODE_FETCH_ANNOTATIONS) {
+            newFeedItems = this.filterAnnotations(feedItems);
```

Comment:
What scenario does this serve? If annotations failed to load, would they not already be absent from the feed items collection?